src/host/test/ledger.cpp

```diff
@@ -1036,7 +1026,201 @@ TEST_CASE("Delete committed file from main directory")
   }
 }
 
-TEST_CASE("Find latest snapshot with corresponding ledger chunk")
+TEST_CASE("Generate and commit snapshots" * doctest::test_suite("snapshot"))
```

Comment:
These are better unit tests of the host snapshotting logic (handling of files, etc.). In particular, this tests that 1.x snapshots can still be read by the 2.x hosts, etc. (also verified by the end-to-end `lts_compatibility` test).